package.json

```diff
@@ -10,14 +10,15 @@
     "debug": "^2.2.0"
   },
   "devDependencies": {
-    "jscs": "^1.13.1",
-    "jshint": "^2.8.0",
+    "eslint": "^2.7.0",
+    "eslint-config-loopback": "^1.0.0",
     "mocha": "^2.2.5",
     "should": "^6.0.3"
   },
   "scripts": {
-    "pretest": "jscs . && jshint .",
-    "test": "mocha"
+    "test": "mocha",
+    "lint": "eslint .",
```

Comment:
put this line above test usually, we should see

```
lint
pretest
test
posttest
```

basically easier to tell which tests are grouped together